kolibri/plugins/learn/assets/src/views/assessment-wrapper/index.vue

```diff
@@ -140,6 +144,10 @@ oriented data synchronization.
         type: Function,
         default: () => Promise.resolve(),
       },
+      hasPrimaryButton: {
```

Comment:
Can we make this prop more clear, e.g. theCheckButtonIsPrimary, well obviously shorter haha